backend/src/question/entities/question.entity.ts

```diff
@@ -5,6 +5,9 @@ export class Question {
     @PrimaryGeneratedColumn({ type: "int8" })
     id: number;
 
+    @Column("text", { nullable: false })
+    name: string;
+
```

Comment:
Better to rename to schema_name so it is clearer. I also think you should set the default value for it to be `public`.

Also a new field called `question_name` which is `UNIQUE NOT NULL` should be needed for displaying the questions leetcode style